modules/internal/src/main/scala/higherkindness/mu/rpc/internal/serviceImpl.scala

```diff
@@ -565,8 +593,9 @@ object serviceImpl {
       }
 
       val operations: List[HttpOperation] = for {
-        d      <- rpcDefs.collect { case x if findAnnotation(x.mods, "http").isDefined => x }
-        args   <- findAnnotation(d.mods, "http").collect({ case Apply(_, args) => args }).toList
+        d <- rpcDefs.collect { case x if findAnnotation(x.mods, "http").isDefined => x }
+        // TODO not sure what the following line is doing, as the result is not used. Is it needed?
+        _      <- findAnnotation(d.mods, "http").collect({ case Apply(_, args) => args }).toList
```

Comment:
might be worth it to log an issue about it so we don't forget about it as easily, wdyt?